class-issue.php

```diff
@@ -904,10 +907,11 @@ function set_bundle_products_array( $bundles, $order){
 		$total_bundle = 0;
 		$total_products = 0;
 		$bundle_products = array();
+		$product_type = '';
```

Comment:
Para corrigir o warning, defini a variável $product_type aqui no início da função.

Mas fiquei em dúvida se faz sentido que essa variável seja acessada posteriormente fora do foreach, na linha 979 do meu código, pois neste momento ela sempre terá o valor que adquiriu na última iteração do foreach. Isso não pode ter alguma consequencia indesejada ou não estar funcionando em algum caso?